documentation/tut/posting_data.rst

```diff
@@ -1,78 +1,33 @@
+
+.. note:: For the purposes of forecasting and scheduling, it is often advisable to use a higher resolution than most metering services keep. For example, while such services might measure every ten seconds, FlexMeasures will usually do its job no less effective if you feed it data with a resolution of five minutes. This will also make the data integration much easier. Keep also in mind that many data sources like weather forecasting or markets can have data resolutions of an hour, anyway.
 
 .. contents:: Table of contents
     :local:
     :depth: 1
 
-Setting up
-----------
-
-Researchers require an admin account to set up a new simulation with a number of assets.
-
-Creating assets and owners
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-New assets can be created through the UI on:
-
-.. code-block:: html
-
-    https://company.flexmeasures.io/assets/new
-
-
-We recommend that researchers choose their own admin account as the asset's owner.
-This way, the simulation will only require refreshing of the access token for the admin account.
-Alternatively, researchers can set up unique accounts for each agent in a multi-agent simulation by creating new owners.
-In this case, access tokens need to be refreshed by each agent separately.
-
-Authentication
-^^^^^^^^^^^^^^
-
-Service usage is only possible with a user authentication token specified in the request header, for example:
-
-.. code-block:: json
-
-    {
-        "Authorization": "<token>"
-    }
-
-The "<token>" can be obtained on your profile after logging in:
-
-.. code-block:: html
-
-    https://company.flexmeasures.io/account
-
-For security reasons, tokens expire after a certain amount of time (see :ref:`_auth`).
-To automate token renewal, use the following POST endpoint:
-
-.. code-block:: html
+Prerequisites
+--------------
 
-    https://company.flexmeasures.io/api/requestAuthToken
+- FlexMeasures needs some structural meta data for data to be understood. For example, for adding weather data we need to define a weather sensor, and what kind of weather sensors there are. You also need a user account. If you host FlexMeasures yourself, you need to add this info first. Head over to :ref:`getting_started`, where these steps are covered, or study our :ref:`cli`.
+- You should be familiar with where to find your API endpoints (see :ref:`api_versions`) and how to authenticate against the API (see :ref:`api_auth`).
 
-Providing applicable user credentials:
+.. note:: For deeper explanations of the data and the meta fields we'll send here, You can always read the :ref:`api_introduction` , e.g. :ref:`signs`, :ref:`resolutions`, :ref:`prognoses` and :ref:`units`.
 
-.. code-block:: json
+.. note:: To address assets and sensors, these tutorials assume entity addresses valid in the namespace ``fm0``. See :ref:`api_introduction` for more explanations. 
```

Comment:
No mention of fm0 there currently. I think this note is dependent on our entity address PR. Maybe you can cut this note for now and paste it as a comment in that PR (to be added back here later)?

Reply:
The way it is going, I believe the entity address PR can land before this one, and then this is correct.

Reply:
Or at least they go out together in v0.5.0